pandas/_libs/tslib.pyx

```diff
@@ -82,12 +79,10 @@ from tslibs.timedeltas cimport cast_from_unit, delta_to_nanoseconds
 from tslibs.timedeltas import Timedelta
 from tslibs.timezones cimport (
     is_utc, is_tzlocal, is_fixed_offset,
-    treat_tz_as_dateutil, treat_tz_as_pytz,
-    get_timezone, get_utcoffset, maybe_get_tz,
+    treat_tz_as_pytz,
+    get_timezone, maybe_get_tz,
     get_dst_info)
-from tslibs.fields import (
-    get_date_name_field, get_start_end_field, get_date_field,
-    build_field_sarray)
```

Comment:
build_field_sarray I think can be completely removed from the codebase

Reply:
It's used in tslibs.resolution.

There are a few funcs scattered around _libs that aren't used anywhere.  Didn't remove them in this PR to keep it uncontroversial.  lib.arrmap and index.get_value_at come to mind.